Dockerfile

```diff
@@ -49,10 +49,12 @@ ENV NGINX_VERSION=$NGINX_VERSION
 ADD nginx-$NGINX_VERSION.tar.gz /tmp/
 
 WORKDIR /tmp
-RUN ./configure && make
+RUN wget https://sourceforge.net/projects/pcre/files/pcre/8.44/pcre-8.44.tar.gz && tar xzvf pcre-8.44.tar.gz
```

Comment:
If you did this on both this command and the one below, the `RUN` command on line 54 wouldn't need to be updated after PCRE or zlib was upgraded:

`RUN wget ... && mkdir -p pcre && tar -xz -C pcre -f pcre-8.44.tar.gz` 

Just a suggestion if you think it would help with maintainability.